components/automate-ui/src/app/entities/rules/rule.reducer.ts

```diff
@@ -88,8 +92,10 @@ export function ruleEntityReducer(
       return set(GET_ALL_STATUS, EntityStatus.loading, state);
 
     case RuleActionTypes.GET_ALL_SUCCESS:
-      return set(GET_ALL_STATUS, EntityStatus.loadingSuccess,
-        ruleEntityAdapter.addAll(action.payload.rules, state));
+      return pipe(
+        set(GET_ALL_STATUS, EntityStatus.loadingSuccess),
+        set(PROJECT_RULES_STATUS, action.payload.status))
```

Comment:
All the back-end work was really to just be able to add the project rules status for each project to the ngrx/store right here.